src/pdfnorm.jl

```diff
@@ -0,0 +1,58 @@
+export pdfL2norm
+
+"""
+    pdfL2norm(d::Distribution)
+
+Return the L2 norm of the probability distribution function `f(x)` of the distribution `d`:
+
+```math
+\\int_{S} f(x)^{2} \\mathrm{d} x
+```
+
+where `S` is the support of `f(x)`.
+"""
+pdfL2norm
+
+pdfL2norm(d::Distribution) = throw(ArgumentError("L2 norm not implemented for $(d)"))
```

Comment:
Is it needed or would be a MethodError sufficient? It seems the error message does not provide any additional information.